### ISSUE TITLE
Order of fine-tuning and testing on the target city in test_model.py

### ISSUE THREAD

**Issue by HQ-LV** [huaxiuyao/MetaST#4]
Thank you for sharing the code. I have a question about `test_model.py`.
Why, in `test_model.py`, is testing done first on the test set of the target city (line28-39) and then fine-tuning on the training set of the target city(line49-60)? Shouldn't it be fine-tuning first and then testing?
